### PR TITLE
(IMAGES-1051) Added windows 2016 french

### PR DESCRIPTION
**Pull request by sebastian-miclea**

### CODE REVIEW

**Comment by sebastian-miclea** [comment]
Related to: https://github.com/puppetlabs/ci-job-configs/pull/5630

File: templates/win/2012r2-fr/x86_64/vars.json [hunk 1]
```diff
@@ -20,4 +20,4 @@
     "InstallationType"  : "Server",
     "ReleaseID"         : "N/A"
 
-}
+}
```
Comment (jcoconnor) [inline templates/win/2012r2-fr/x86_64/vars.json]: There is a missing end of line here.
Suggest you make sure your editor settings are set to do this by default - see https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files

File: templates/win/2016-fr/x86_64/vars.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    "boot_command"      : "<enter><wait><enter><wait><enter><wait><enter><wait><enter><wait><enter><wait><enter>",
+
+    "CurrentVersion"    : "6.3",
+    "ProductName"       : "Windows Server 2016 Standard",
+    "EditionID"         : "ServerStandard",
+    "InstallationType"  : "Server",
+    "ReleaseID"         : "1607",
+
+    "vmware_vsphere_nocm_required_modules": ""
+}
```
Comment (jcoconnor) [inline templates/win/2016-fr/x86_64/vars.json]: Missing EOL

File: templates/win/2016-fr/x86_64/vars.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    "iso_checksum"      : "da08c746118bc6fc94ee6b5021530357",
+    "boot_command"      : "<enter><wait><enter><wait><enter><wait><enter><wait><enter><wait><enter><wait><enter>",
+
+    "CurrentVersion"    : "6.3",
+    "ProductName"       : "Windows Server 2016 Standard",
+    "EditionID"         : "ServerStandard",
+    "InstallationType"  : "Server",
+    "ReleaseID"         : "1607",
+
+    "vmware_vsphere_nocm_required_modules": ""
```
Comment (jcoconnor) [inline templates/win/2016-fr/x86_64/vars.json]: What is this for ?

File: templates/win/2016/x86_64/vars.json [hunk 4]
```diff
@@ -18,4 +18,4 @@
     "InstallationType"  : "Server",
     "ReleaseID"         : "1607"
 
-}
+}
```
Comment (jcoconnor) [inline templates/win/2016/x86_64/vars.json]: Missing EOL

File: templates/win/common/virtualbox.vagrant.cygwin.json [hunk 5]
```diff
@@ -112,10 +112,7 @@
           "C:/Packer/Scripts/test-packerbuild -TestPhase windows-update"
         ]
     },
-    {
-        "type": "powershell",
-        "script" : "../../common/scripts/provisioners/test-windows-update.ps1"
-    },
+
```
Comment (jcoconnor) [inline templates/win/common/virtualbox.vagrant.cygwin.json]: Vagrant scripts are bit behind but this is likely going to change in the future as I think we should adapt the ```platform9``` build sequence to do a vagrant second stage.

File: templates/win/common/scripts/bootstrap/bootstrap-packerbuild.ps1 [hunk 6]
```diff
@@ -35,7 +35,7 @@ if (-not (Test-Path "$PackerScripts\windows-env.ps1" )) {
 # Create Scheduled Task so this repeatedly until we have finished.
 if (-not (Test-Path "$PackerLogs\BootstrapSchedTask.installed")) {
   Write-Output "Create Bootstrap Scheduled Task"
-  schtasks /create /tn PackerBootstrap /rl HIGHEST /ru "$AdminUser" /RP "$AdminPassword" /F /SC ONSTART /DELAY 0000:20 /TR 'cmd /c c:\WINDOWS\system32\WindowsPowerShell\v1.0\powershell.exe -sta -WindowStyle Hidden -ExecutionPolicy Bypass -NonInteractive -NoProfile -File A:\bootstrap-packerbuild.ps1 >> C:\Packer\Logs\bootstrap-packerbuild.log'
+  schtasks /create /tn PackerBootstrap /rl HIGHEST /ru "$AdminUsername" /RP "$AdminPassword" /F /SC ONSTART /DELAY 0000:20 /TR 'cmd /c c:\WINDOWS\system32\WindowsPowerShell\v1.0\powershell.exe -sta -WindowStyle Hidden -ExecutionPolicy Bypass -NonInteractive -NoProfile -File A:\bootstrap-packerbuild.ps1 >> C:\Packer\Logs\bootstrap-packerbuild.log'
```
Comment (jcoconnor) [inline templates/win/common/scripts/bootstrap/bootstrap-packerbuild.ps1]: 👍

File: templates/win/2016-fr/x86_64/vars.json (hunk 3 above)
Comment (sebastian-miclea) [reply templates/win/2016-fr/x86_64/vars.json]: I was left behind from some trial runs, I removed it.

**Pull request merged**